frontend/src/app/offers/offers.component.html

```diff
@@ -0,0 +1,24 @@
+<section class="container">
+  <div class="row">
+    <article *ngFor="let offer of offers" class="offer col-sm-6 col-md-4">
+      <a href="{{ offer.url }}" class="offer__img"
+         [ngStyle]="{'background-image': offer.image ? 'url(/' + offer.image + ')' : 'url(/media/)'}"></a>
+      <h2 class="offer__title">
+        <a href="{{ offer.url }}" class="offer__link">{{ offer.title }}</a>
+      </h2>
+      <div class="offer__caption">
+        <div class="offer__row">
+          <span aria-hidden="true" class="offer__glyphicon glyphicon glyphicon-map-marker"></span>
+          <span>{{ offer.location }}</span>
+        </div>
+        <section class="offer__row">
+          <span aria-hidden="true" class="offer__glyphicon glyphicon glyphicon-time"></span>
+          <span> Od {{ offer.startedAt ? offer.startedAt : 'teraz' }} </span>
+          &nbsp;
+          <span> do {{ offer.finishedAt ? offer.finishedAt : 'ustalenia' }}</span>
+        </section>
+        <div class="offer__organizer">Organizator:</div>
```

Comment:
It looks that it need a lot of love, but e.g. it's missing organisation name here.